android/src/main/java/com/twiliovoicereactnative/NotificationUtility.java

```diff
@@ -293,41 +298,67 @@ public static Notification createWakeupAppNotification(String callSid, int notif
   }
 
   @TargetApi(Build.VERSION_CODES.O)
-  private static int getChannelImportance(final String voiceChannel) {
-    if (Objects.equals(voiceChannel, Constants.VOICE_CHANNEL_HIGH_IMPORTANCE)) {
-      return NotificationManager.IMPORTANCE_HIGH;
-    } else if (Objects.equals(voiceChannel, Constants.VOICE_CHANNEL_DEFAULT_IMPORTANCE)) {
-      return NotificationManager.IMPORTANCE_DEFAULT;
-    } else if (Objects.equals(voiceChannel, Constants.VOICE_CHANNEL_LOW_IMPORTANCE)) {
-      return NotificationManager.IMPORTANCE_LOW;
+  public static void createNotificationChannels(Context context) {
+    NotificationManager notificationManager =
+      (NotificationManager)context.getSystemService(Context.NOTIFICATION_SERVICE);
+    notificationManager.createNotificationChannelGroup(
+      new NotificationChannelGroup(Constants.VOICE_CHANNEL_GROUP, "Twilio Voice"));
+    for (String channelId:
+      new String[]{
+        VOICE_CHANNEL_HIGH_IMPORTANCE,
+        VOICE_CHANNEL_DEFAULT_IMPORTANCE,
+        VOICE_CHANNEL_LOW_IMPORTANCE}) {
+      notificationManager.createNotificationChannel(
+        createNotificationChannel(context, channelId));
     }
-    return NotificationManager.IMPORTANCE_DEFAULT;
   }
+
   @TargetApi(Build.VERSION_CODES.O)
-  private static String getChannel(Context context, final String voiceChannelId) {
+  public static void destroyNotificationChannels(Context context) {
+    NotificationManager notificationManager =
+      (NotificationManager)context.getSystemService(Context.NOTIFICATION_SERVICE);
+    notificationManager.deleteNotificationChannelGroup(Constants.VOICE_CHANNEL_GROUP);
+  }
+
+  @TargetApi(Build.VERSION_CODES.O)
+  private static NotificationChannel createNotificationChannel(Context context,
+                                                               final String voiceChannelId) {
+    final int notificaitonImportance = getChannelImportance(voiceChannelId);
```

Comment:
```suggestion
    final int notificationImportance = getChannelImportance(voiceChannelId);
```

Reply:
I think there are other references to this variable that are now invalid if we just merge this suggested change. @kbagchiGWC